src/rpc/blockchain.cpp

```diff
@@ -2219,7 +2223,12 @@ static RPCHelpMan savemempool()
         throw JSONRPCError(RPC_MISC_ERROR, "Unable to dump mempool to disk");
     }
 
-    return NullUniValue;
+    NodeContext& node = EnsureAnyNodeContext(request.context);
```

Comment:
Completely hypothetical scenario at the moment, but if there is no node context, we probably shouldn't throw an error here since the dump has presumably already succeeded.

Reply:
Done in 1b55103